README.md

```diff
@@ -114,16 +126,46 @@ make server-init QUIPUCORDS_SERVER_PASSWORD="SuperAdmin1"
 Both of the above commands create a superuser with name `admin` and password of `SuperAdmin1`.
 
 ## Running the Server
+
+### Starting the API Server
+To start the server with API functionality only:
 ```
 make server-static
 make serve
 ```
-To log in to the server, you must connect to http://127.0.0.1:8000 and provide the superuser credentials stated above.
+This will make the API accessible at http://127.0.0.1:8000. 
 
-After logging in, you can change the password and also go to some browsable APIs such as http://127.0.0.1:8000/api/v1/credentials/.
 To use the command line interface, you can configure access to the server by entering `qpc server config`. You can then log in by using `qpc server login`.
 
-If you want to also use the UI, please refer to [quipucords-ui](https://github.com/quipucords/quipucords-ui) for further information.
+### Starting the Celery Worker
+
+Open a separate terminal and run the Celery worker process:
+```
+make celery-worker
+```
+This starts a Celery worker for handling background tasks and asynchronous operations. The Celery worker must remain running alongside the server for proper operation of scheduled tasks and scans.
+
+### Running with UI (Optional)
+If you want to use the UI, you need to install [quipucords-ui](https://github.com/quipucords/quipucords-ui) in a directory at the same level as quipucords. Follow these steps:
+
+1. Clone the **quipucords-ui** repository next to your **quipucords** directory.
+2. Navigate to the **quipucords-ui** directory.
+3. Follow the **installation instructions** in the [quipucords-ui repository](https://github.com/quipucords/quipucords-ui).
+4. Set the following environment variables according to the server configuration:
+```
+export QUIPUCORDS_SERVER_PROTOCOL
+export QUIPUCORDS_SERVER_HOST
+export QUIPUCORDS_SERVER_PORT
+```
+5. Start the UI with:
+```
+npm run start:using-server
+```
+This will start a proxy to communicate with the server and open the UI in your browser, typically at http://localhost:3000. The UI provides a more user-friendly way to interact with the API and manage your quipucords server.
+
+You can log in using the **credentials provided during server setup** (e.g., **Username:** `admin`, **Password:** `SuperAdmin1` unless changed).
```

Comment:
Do we have an "default password"? I thought we completely removed this possibility per prosec request 🤔

Reply:
[Earlier section of README](https://github.com/quipucords/quipucords?tab=readme-ov-file#initializing-the-server) tells you to run: `make server-init QUIPUCORDS_SERVER_PASSWORD="SuperAdmin1"`

I guess this is what this change refers to.

Reply:
Thanks, @mirekdlugosz ;)

Reply:
We could go out with a limb and remove this as well, but this is a discussion out of scope of this PR. 

I'm just going to approve it as is